src/test/unit/math/SparseBinaryMatrixTest.cpp

```diff
@@ -24,17 +24,16 @@
 #include <utility>
 #include <vector>
 
-#include <capnp/message.h>
-#include <capnp/serialize.h>
 #include <gtest/gtest.h>
-#include <kj/std/iostream.h>
 
 #include <nupic/math/SparseBinaryMatrix.hpp>
-#include <nupic/proto/SparseBinaryMatrixProto.capnp.h>
 #include <nupic/types/Types.h>
 
 using namespace nupic;
 
+
+/************************ToDo: replace with streaming serialization
+
```

Comment:
maybe better keep the tests there, and just fix them later. Now it's possible to forget the test commented out.